packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -111,6 +113,36 @@ function MapsIndoorsMap(props) {
     )
 }
 
+Sentry.init({
+    dsn: "https://0ee7fa162023d958c96db25e99c8ff6c@o351128.ingest.sentry.io/4506851619831808",
+    // Set environment to localhost if the url includes it. Otherwise, set to production.
+    environment: window.location.hostname === 'localhost' ? 'localhost' : 'production',
+    integrations: [
+        // See docs for support of different versions of variation of react router
```

Comment:
I still think a comment should be made about why we use React Router stuff here even though we don't use the React Router in the project otherwise.

Reply:
@ammapspeople I added a comment in the code now 👍 (I thought you just wanted one in GitHub 🙃)

Reply:
Perfect 👍